qemu/tests/uefishell.py

```diff
@@ -76,6 +80,24 @@ def copy_uefishell(self):
             process.system(cp_command)
         return uefishell_dst_path
 
+    def copy_secureboot_pk_kek(self):
+        """
+        Copy SecureBootPkKek1.oemstr
+        :return SecureBootPkKek1.oemstr path
+        """
+        pk_kek_filepath = data_dir.get_deps_dir("edk2")
+        pk_kek_filename = "SecureBootPkKek1.oemstr"
+        pk_kek_src_path = utils_misc.get_path(pk_kek_filepath,
+                                              pk_kek_filename)
+        pk_kek_dst_path = "images/%s" % pk_kek_filename
+        pk_kek_dst_path = utils_misc.get_path(data_dir.get_data_dir(),
+                                              pk_kek_dst_path)
+        if not os.path.exists(pk_kek_dst_path):
+            cp_command = "cp -f %s %s" % (
```

Comment:
we copy a new file before test, we remove it after test, that might be better.

Reply:
I will follow function var_copy().
copy a new SecureBootPkKek1.oemstr file for a cleaner environment to test.

Reply:
Actually, I found the following files existed after testing:
UefiShell.iso
SecureBootPkKek1.oemstr
rhel810-64-virtio-scsi.raw.fd

Reply:
it's better to remove SecureBootPkKek1.oemstr; for the others, we can open a bug to remove them in future; what's your opinion?

Reply:
I think it is not important. The related files \"UefiShell.iso\", \"rhel810-64-virtio-scsi.raw.fd\" and \"SecureBootPkKek1.oemstr\" will be overwrote every time when running automation cases.
As far as I know, don't remove them is better to debug code manually. I think we can open a bug to remove all of them together in future. Do you agree?

Reply:
OK, that would be fine, please open a bug to track it.